eta-company-calendar-backend/src/main/java/hu/flowacademy/companycalendar/controller/MeetingResource.java

```diff
@@ -0,0 +1,46 @@
+package hu.flowacademy.companycalendar.controller;
+
+import hu.flowacademy.companycalendar.model.Meeting;
+import hu.flowacademy.companycalendar.model.dto.MeetingDTO;
+import hu.flowacademy.companycalendar.service.MeetingService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/meetings")
+@AllArgsConstructor
+public class MeetingResource {
+
+    private final MeetingService meetingService;
+
+    @GetMapping("")
+    public List<Meeting> getAll() {
+        return meetingService.findAll();
+    }
+
+    @GetMapping("/{id}")
+    public Meeting getOne(@PathVariable Long id) {
+        return meetingService.findOne(id);
+    }
+
+    @PostMapping("/users/{id}")
```

Comment:
Itt miért van `/users`? nem lenne elég csak a `/{id}`?

Reply:
Elsőre úgy gondoltam, így egyértelműbb, hogy itt UserId-t vár az endpoint, más oka nincs - átírom. Viszont felmerült, hogy jó-e így, hogy UserID-t PathVariable-ként kapja a metódus, vagy az is a RequestBody-ban legyen?